specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/networkWatcher.json

```diff
@@ -2827,6 +2827,10 @@
         "protocolConfiguration": {
           "$ref": "#/definitions/ProtocolConfiguration",
           "description": "Configuration of the protocol."
+        },
+        "preferredIPVersion": {
```

Comment:
Adding a writable property in a shipped API version is considered a breaking change as it can impact GET->PUT pipeline. Is this a bug fix or a new property addition?

Reply:
This is a newly added property, and this property is nullable.

Reply:
@KrisBash @junyezhu also, AFAIK change with this property is live yet and this PR is to get initial design review. If it's live, this change should go into release branch

Reply:
@junyezhu - any writable property in an existing api version is considered breaking, as a user with a new sdk could set the property. Afterwards, a user with an older SDK could do a GET and PUT to update the resource, and omit the new property. What is the service behavior in that case?

Is this API live?

Reply:
@KrisBash ConnectivityCheck is live, but this feature is a one time operation. There is no resource stored for customer to update using GET and PUT. So even though behavior different between existing and new api version, it won't be a breaking change.